carl/logic/copd.py

```diff
@@ -19,6 +19,25 @@
     code="CNICS.COPD2021.11.001",
     display="COPD PRO group member")
 
+CNICS_IDENTIFIER_SYSTEM = "https://cnics.cirg.washington.edu/"
+
+
+def patient_canonical_identifier(patient_id):
+    """Return system|value identifier if patient has one for preferred system"""
+    url = f"{FHIR_SERVER_URL}Patient/{patient_id}"
+    response = requests.get(url)
+    if has_app_context():
+        current_app.logger.debug(f"HAPI GET: {response.url}")
```

Comment:
Shouldn't this be the _requested_ URL (for GET)?
```suggestion
        current_app.logger.debug(f"HAPI GET: {request.url}")
```

Reply:
it is producing the correct string in the log - you'll note there isn't a `request.url` but rather we're using (with an s) the `requests` lib, which returns the full URL used (might include variable interpolation, etc.)